code/controllers/configuration/entries/game_options.dm

```diff
@@ -372,7 +372,7 @@
 	min_val = 0
 
 /datum/config_entry/number/space_budget
-	config_entry_value = 16
+	config_entry_value = 60
```

Comment:
60 ruins

Reply:
I change the cost value of ruins.

Unless it doesn't account the cost at all.

Reply:
Pretty sure all ruins count as 1 as found in bluespace exploration

Reply:
ah you changed the cost, check its actually used

Reply:
Ok. Also, why won't it compile?

Reply:
I checked, and they use the same code as lavaland ruins, so cost is taken into accound.

It was just that all cost was at 1.